test/v1alpha1/cancel_test.go

```diff
@@ -115,7 +115,7 @@ spec:
 			}
 
 			t.Logf("Waiting for PipelineRun %s in namespace %s to be cancelled", pipelineRunName, namespace)
-			if err := WaitForPipelineRunState(ctx, c, pipelineRunName, pipelineRunTimeout, FailedWithReason("PipelineRunCancelled", pipelineRunName), "PipelineRunCancelled"); err != nil {
+			if err := WaitForPipelineRunState(ctx, c, pipelineRunName, pipelineRunTimeout, FailedWithReason("Cancelled", pipelineRunName), "PipelineRunCancelled"); err != nil {
```

Comment:
I believe v1alpha1 should continue using `PipelineRunCancelled` 🤔 

(v1alpha1 is read-only, we don't make change to it)